praxis/layers/models.py

```diff
@@ -98,6 +99,8 @@ def compute_xent_loss_helper(
       example weights from the input `eval_sample_weights` or not. When enabled,
       these per-example weights will be merged with the per token
       `input_batch.weights`.
+    report_strict_acc: Whether to report strict accuracy. Used for eval on 
```

Comment:
Could you extend this comment? In the case of Lambada, I guess you're only expecting a single word. But what this metrics seems to do is compute a per sequence exact match at least for the non-zero weighted tokens.